src/App.js

```diff
@@ -1,17 +1,39 @@
-import './App.css';
+import "./App.css";
 
+import { getUsers } from "./utils/api";
+import { useEffect, useState } from "react";
+import { Reviews } from "./elements/Reviews";
+import { Link, useLocation, Routes, Route } from "react-router-dom";
 
 function App() {
+  const url = useLocation();
+  const [user, setUser] = useState(undefined);
+  
+
+  useEffect(() => {
+    getUsers().then((users) => {
+      if (users.length !== 0) setUser(users[0]);
+    });
+  }, []);
+
   return (
     <div className="App">
       <header>
-        Haha
+        <h1>Table of Tops</h1>
       </header>
       <nav>
-        Haha
+        {user ? user.name : "Loading User"}
+        <Link to="/">
+          <button disabled={url.pathname === "/"}>Reviews</button>
```

Comment:
You don't need to put buttons inside Links as they're already clickable and clicking on the link you're already on has no effect as it's default behaviour. If you doing this for styling reasons you can change the class based on the this condition or use a [NavLink component](https://reactrouter.com/en/main/components/nav-link) to achieve this result